data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/modify_unit_recall_cost.cfg

```diff
@@ -0,0 +1,83 @@
+#textdomain wesnoth-test
+# Actions:
+# Create Charlie, belonging to side 2, on the recall list
+# Increase Charlie's recall cost
+# Recall Charlie next to his side's leader.
+##
+# Expected end state:
+# Charlie is on the map next to side 2's leader.
+# Charlie's recall cost has increased by 10
+#####
+{GENERIC_UNIT_TEST "modify_unit_recall_cost" (
+    [event]
+        name= start
+        # Create a unit on the recall list of side 2
+        [unit]
+            x=recall
+            y=recall
+            type=Orcish Grunt
+            side=2
+            id=Charlie
+            name=_"Charlie"
+            role=TestSubject
+            canrecruit=no
+            recall_cost=10
+        [/unit]
+
+        # Increase Charlie's recall cost
+        [modify_unit]
+            [filter]
+                id=Charlie
+                side=2
+            [/filter]
+            [object]
+                [effect]
+                    apply_to="recall_cost"
+                    increase=10
+                [/effect]
+            [/object]
+        [/modify_unit]
+
+        # Check that Charlie is still on the recall list, that modifying him
+        # hasn't moved him back to the map.
+        {ASSERT (
+            [not]
+                [have_unit]
+                    id=Charlie
+                    search_recall_list=no
+                [/have_unit]
+            [/not]
+        )}
+
+        # Check that Charlie's recall cost has increased
+        [store_unit]
+            [filter]
+                id=Charlie
+            [/filter]
+            variable=temp
+        [/store_unit]
+
+        {ASSERT (
+            {VARIABLE_CONDITIONAL temp.recall_cost equals 20}
```

Comment:
Starting with an initial cost of 10 and then increasing by 10 means the expected value is the same as the default value. Please change one of those numbers, so that this is testing that the unit has an individual recall cost.

Reply:
Got it, good catch